roles/runtime/tasks/crio.yaml

```diff
@@ -0,0 +1,48 @@
+---
+- name: Install CRI-O and dependencies.
+  yum:
+    name:
+      - conmon
+    state: present
+    disable_gpg_check: true
+
+# TODO: Add support to generate a nix-build for ppc64le or add actions to generate a binary for cri-o
+- name: Download CRI-O.
+  unarchive:
+    src: "https://github.com/kishen-v/crio-ppc64le/raw/main/crio-1.26.tar.gz"
```

Comment:
Currently cloned from a user-based repository. Please let me know if having it moved to a repository under`ppc64le-cloud` is advised.

Reply:
yep, lets move to ppc64le-cloud org and modify the code accordingly.

Reply:
Addressed.